tox.ini

```diff
@@ -1,23 +1,48 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
 [tox]
-minversion=3.6.0
-envlist = flake8,py3.9,py3.8,py3.7,py3.6
+minversion = 3.10
```

Comment:
Should `minversion` be `3.10`, or left at `3.6` ?

Reply:
why lower? 
we are installing tox 3.24 via poetry anyways.

Reply:
RTD: https://tox.wiki/en/latest/config.html#conf-minversion